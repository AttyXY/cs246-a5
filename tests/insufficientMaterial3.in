setup 
+ K e1
+ n h1
+ k e8
done
