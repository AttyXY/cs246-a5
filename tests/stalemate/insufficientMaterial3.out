WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

WELCOME TO SETUP MODE!
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- + [PieceType] [Coordinate] (to add piece)
- - [Coordinate] (to remove piece)
- = [Colour] (to set turn)
- done (to exit setup mode)

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--n
   abcdefgh

8  ----k---
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--n
   abcdefgh

PLEASE ENTER THE FOLLOWING COMMANDS TO BEGIN:
- game [player1] [player2]

STARTING GAME!

8  ----k---
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--n
   abcdefgh

Stalemate!

PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- setup (to create new setup)
- game (to continue with previous setup)
- quit (to exit)

THANKS FOR PLAYING CHESS!
Final Score:
White: 0.5
Black: 0.5
